packages/notification-center/src/i18n/languages/pl.ts

```diff
@@ -1,11 +1,11 @@
-import { ITranslationEntry } from "../lang";
+import { ITranslationEntry } from '../lang';
 
 export const PL: ITranslationEntry = {
-    translations: {
-      notifications: 'Notyfikacje',
-      markAllAsRead: 'Oznacz wszystkie jako przeczytane',
-      poweredBy: 'Napędzany przez',
-      settings: 'Ustawienia'
-    },
-    lang: 'pl',
-  };
\ No newline at end of file
+  translations: {
+    notifications: 'Notyfikacje',
+    markAllAsRead: 'Oznacz wszystkie jako przeczytane',
+    poweredBy: 'Napędzany przez',
+    settings: 'Ustawienia',
+  },
+  lang: 'pl',
+};
```

Comment:
This change seems to not belong to the target of this PR. 😓

Reply:
Looks like prettier things, but that's actually the correct implementation. So I think we can leave it here